doc/source/examples/sphinx-autoapi.rst

```diff
@@ -0,0 +1,13 @@
+.. _sphinx-autoapi:
+
+Sphinx autoapi
+==============
+
+This section describes ansys-sphinx-theme endpoints, their capabilities, and how
+to interact with them programmatically.
```

Comment:
```suggestion
This section describes how to use Python to interact programmatically with the Ansys Sphinx Theme.
```
I'm not sure this is what you want for the introduction. It seems like it should be about Sphinx AutoAPI--not our theme?